notebook2/notebook/notebook.py

```diff
@@ -473,26 +420,28 @@ def wait_websocket(ws):
     if notebook is None:
         return
 
-    pod_name = notebook['pod_name']
-    svc_name = notebook['svc_name']
+    pod_uuid = notebook['pod_uuid']
+
+    url = f'{external_url_for("")}instance-ready/{pod_uuid}/'
```

Comment:
`external_url_for(f'instance-ready/{pod_uuid}')`

Reply:
Is that any better?